drivers/media/i2c/lt6911uxc.c

```diff
@@ -734,6 +744,43 @@ static int lt6911uxc_dv_timings_cap(struct v4l2_subdev *sd,
 	return 0;
 }
 
+static int lt6911uxc_get_mbus_config(struct v4l2_subdev *sd,
+				unsigned int pad,
+				struct v4l2_mbus_config *cfg)
+{
+	struct lt6911uxc_state *state = to_state(sd);
+	const u32 mask = V4L2_MBUS_CSI2_LANE_MASK;
+
+	cfg->type = V4L2_MBUS_CSI2_DPHY;
+	cfg->flags = (state->csi_lanes_in_use << __ffs(mask)) & mask;
+
+	/* In DT mode, only report the number of active lanes */
+	if (sd->dev->of_node)
+		return 0;
+
+	/* Support for non-continuous CSI-2 clock is missing in pdate mode */
+	cfg->flags |= V4L2_MBUS_CSI2_CONTINUOUS_CLOCK;
```

Comment:
You've pushed this as a PR on a 6.0 branch.
5.18 and above have commit https://github.com/torvalds/linux/commit/6a7bdd89f50d399dd02847e6f398d408b086df50 which totally changes ```struct v4l2_mbus_config```. I don't believe this branch will even compile.